services/storage/tests/unit/test_async_jobs.py

```diff
@@ -52,22 +53,24 @@
 async def rpc_sync_job(
     app: FastAPI, *, job_id_data: AsyncJobNameData, **kwargs: Any
 ) -> AsyncJobGet:
-    task_uuid = await get_celery_client(app).send_task(
-        sync_job.__name__, task_context=job_id_data.model_dump(), **kwargs
+    task_name = sync_job.__name__
+    task_uuid = await get_celery_client(app).submit_task(
+        TaskMetadata(name=task_name), task_context=job_id_data.model_dump(), **kwargs
     )
 
-    return AsyncJobGet(job_id=task_uuid)
+    return AsyncJobGet(job_id=task_uuid, job_name=task_name)
 
 
 @router.expose()
 async def rpc_async_job(
     app: FastAPI, *, job_id_data: AsyncJobNameData, **kwargs: Any
 ) -> AsyncJobGet:
-    task_uuid = await get_celery_client(app).send_task(
-        async_job.__name__, task_context=job_id_data.model_dump(), **kwargs
+    task_name = async_job.__name__
+    task_uuid = await get_celery_client(app).submit_task(
+        TaskMetadata(name=task_name), task_context=job_id_data.model_dump(), **kwargs
```

Comment:
nice
NOTE:  since you have validators on obht sides, dont you prefer to `job_id_data.model_dump(exclude_unset=True)` ?